File: bialydunajec-news/application/src/main/kotlin/org/bialydunajec/news/application/dto/CampNewsPageDto.kt
package org.bialydunajec.news.application.dto

data class CampNewsPageDto(
        val content: Collection<CampNewsDto>,
        val paging: NewsPagingDto
)

data class NewsPagingDto(
        val cursors: PagingCursorsDto?
)

data class PagingCursorsDto(
        val before: String?,
        val after: String?
)

File: bialydunajec-adminpanel-frontend/projects/bialydunajec-main/src/app/camp-news/service/rest/dto/camp-news-page.dto.ts
export class CampNewsPageDto {
  content: CampNewsDto[];
  paging: PagingCursorsDto;
}

export class NewsPagingDto {
  cursors: PagingCursorsDto;
}

export class PagingCursorsDto {
  before: string;
  after: string;
}

export class CampNewsDto {
  id: string;
  message: string;
  picture: NewsPictureDto;
  createdDate: Date;
}

export class NewsPictureDto {
  url: string;
  name: string;
  description: string;
}
